common/app/views/fragments/omnitureScript.scala.html

```diff
@@ -1,20 +1,12 @@
 @(item: Option[model.MetaData])
 
 @import conf.Static
-@import conf.Configuration
 @import views.support.OmnitureAnalyticsAccount
 
 @item.map{ item =>
     <script>
         @* must always be set before the Omniture file is parsed *@
         window.s_account = '@OmnitureAnalyticsAccount(item)';
+        @Html(Static.js.omnitureJs)
```

Comment:
If the Option is a `None`, we don't want to include the Omniture library, or do we?